zeppelin/hdi_add_zeppelin.sh

```diff
@@ -3,127 +3,57 @@
 #----------------------------------------------------------------
 # THIS SCRIPT ADDS INSTALLS AND STARTS ZEPPELIN ON HBASE CLUSTER 
 #----------------------------------------------------------------
+#Import helper module
+wget -O /tmp/HDInsightUtilities-v01.sh -q https://hdiconfigactions.blob.core.windows.net/linuxconfigactionmodulev01/HDInsightUtilities-v01.sh && source /tmp/HDInsightUtilities-v01.sh && rm -f /tmp/HDInsightUtilities-v01.sh
```

Comment:
are you using any of these utilities? It's better to not take dependencies on this if you don't need it.

Reply:
sorry forgot to remove this; initially i was planning on using this but not anymore

Reply:
Readded this now that we are using the utilities